docker-entrypoint-funcs.sh

```diff
@@ -347,6 +347,16 @@ edbdocker_setup_env() {
 
   export EDGEDB_SERVER_RUNSTATE_DIR="${EDGEDB_SERVER_RUNSTATE_DIR:-/run/edgedb}"
 
+  if [ -n "${EDGEDB_SERVER_ALLOW_INSECURE_HTTP_CLIENTS:-}" ]; then
+    if [ -z "${EDGEDB_SERVER_HTTP_ENDPOINT_SECURITY:-}" ]; then
+      export EDGEDB_SERVER_HTTP_ENDPOINT_SECURITY="optional"
+    elif [ "${EDGEDB_SERVER_HTTP_ENDPOINT_SECURITY:-}" = "optional" ]; then
+      :
+    else
+      edbdocker_die "ERROR: EDGEDB_SERVER_ALLOW_INSECURE_HTTP_CLIENTS and EDGEDB_SERVER_HTTP_ENDPOINT_SECURITY are mutually exclusive, but both are set"
```

Comment:
This is nice but unfortunately we are setting both env vars in the CLI for easier compatibility support.

Reply:
Ah nvm, `EDGEDB_SERVER_HTTP_ENDPOINT_SECURITY=optional` is skipped.